tests/test_operator.py

```diff
@@ -30,8 +32,8 @@ def setUp(self):
             "hostess_docker_image": "ocadotechnology/mirror-hostess",
             "hostess_docker_tag": None,
             "image_pull_secrets": None,
-            "docker_certificate_secret": VALID_SECRET,
```

Comment:
This was slightly confusing. `VALID_SECRET` was the contents of a username and password secrets manifest. But what this environment variable wants to be set to is the name of some already existing secret.